modules/installation-launching-installer.adoc

```diff
@@ -49,8 +49,12 @@ endif::[]
 When the cluster deployment completes, directions for accessing your cluster,
 including a link to its web console and credentials for the `kubeadmin` user,
 display in your terminal.
++
+[IMPORTANT]
+====
+You must not delete the installation media or the files that the installation
+program creates. Both are required to delete the cluster.
+====
```

Comment:
What is the _installation media_? Is that the same as the _installation program_ in uninstall?

Reply:
The same question here. "the installation media" -> "the installer binary"?

Reply:
Yes. I'll update it.
Based on our style guide, we shouldn't call it the "installation binary."
> binary- Do not use as a noun. Use only as an adjective, as in “binary file.”

Reply:
It's the installation program. I'll update it.